### ISSUE TITLE
render issues

### ISSUE THREAD

**Issue by le1smo** [queensferryme/hugo-theme-texify#22]
Thanks for your pretty theme.

There is a render issue when use a different customer font in Safari and mobile.
<img width="693" alt="image" src="https://user-images.githubusercontent.com/76763517/166912379-920aa0a9-d46c-4bc8-ad7d-23d27d592319.png">

No problem in Chrome.

<img width="693" alt="image" src="https://user-images.githubusercontent.com/76763517/166912424-93f065f3-1632-4f2c-975b-7aa0483da700.png">

custom.css
```css
@import url('https://cdn.jsdelivr.net/npm/lxgw-wenkai-webfont@1.6.0/style.css');

#wrapper {
    font-family: 'LXGW WenKai', "Times New Roman", serif
}
```

**Comment by queensferryme** [comment]
No problem in Firefox here too. Am looking into the issue.

**Comment by queensferryme** [comment]
Please update the theme and see if it is fixed.

**Comment by le1smo** [comment]
Thanks for your response, it works.

But there are two places are remaining not rendered properly, can you take a look?

<img width="283" alt="image" src="https://user-images.githubusercontent.com/76763517/167233176-e16cd30f-109a-4fdd-8545-e8832ee7dacd.png">

The title should be rendered in one line.

<img width="329" alt="image" src="https://user-images.githubusercontent.com/76763517/167233215-95ae2dc3-18be-4f63-b924-72690f3b405c.png">

Same problem.

**Comment by queensferryme** [comment]
题外话：霞鹭文楷的粗体和常规体区分度不明显，有时候不仔细查看都无法发现某段文字有加粗。因此个人并不建议你使用这种字体。

**Comment by queensferryme** [comment]
@justlongfei 

**TLDR:**
use `font-family: 'LXGW WenKai'` instead of `font-family: 'LXGW WenKai', 'Times New Roman', serif`.

**Long explanation:**

I believe this bug belongs to Safari or LXGW Wenkai. But I don't have the time to dig into the root cause right now.
I can't fix this bug, but you can avoid the bug with `font-family: 'LXGW WenKai'` instead of `font-family: 'LXGW WenKai', 'Times New Roman', serif` (somehow, I don't know why).
A minimal reproducible example:
```html
<!DOCTYPE html>
<html lang="en">

<head>
    <style>
        @import url('https://cdn.jsdelivr.net/npm/lxgw-wenkai-webfont@1.6.0/style.css');

        #wrapper {
            display: flex;
            font-family: 'LXGW WenKai', serif;
        }
    </style>
</head>

<body>
    <div id="wrapper">
        <h1>
            <a href="/">Any Text with Word-Breaks</a>
        </h1>
        <h1>
            <a href="/">OK_When_No_Word_Breaks</a>
        </h1>
    </div>
</body>

</html>
```
Renders like this on Safari:
![图片](https://user-images.githubusercontent.com/41495669/167255179-7ed48493-7d13-4155-892e-905c945558b5.png)

**Comment by justlongfei** [comment]
https://github.com/rsms/inter/issues/321#issuecomment-701576648

**Comment by le1smo** [comment]
similar issue https://github.com/rsms/inter/issues/321

**Comment by le1smo** [comment]
<img width="1355" alt="image" src="https://user-images.githubusercontent.com/76763517/167261797-662b4878-855f-407f-a0da-c142c579c64f.png">

**Comment by le1smo** [comment]
I don't know, maybe really hit WebKit's bug.

**Comment by benjaminv** [comment]
> Renders like this on Safari: ![图片](https://user-images.githubusercontent.com/41495669/167255179-7ed48493-7d13-4155-892e-905c945558b5.png)

It appears to be some bug introduced by the cdn stylesheet however I did not study this through.
To fix, do something like following to the H1 tag, for instance apply flex:auto, to make them share the full width of flex container therefore reduce chances of getting word-wrapped. Its inner a tag natively has no width creates a log of problems among different browsers. 

```html
#wrapper {
            display: flex;
            font-family: 'LXGW WenKai', serif;
            background-color: blueviolet;
            /* justify-content: space-around; */  /* not needed if inner elements have flex-grow and basis */
            /* white-space: nowrap; */  /* result unnecessary overflow */
        }
        h1:first-of-type {
            background-color: lightblue;
        }
        h1:last-of-type {
            background-color: lightgreen;
        }
        h1 {
            flex: auto;
            text-align: center;
        }
```

**Comment by lxgw** [comment]
> 题外话：霞鹭文楷的粗体和常规体区分度不明显，有时候不仔细查看都无法发现某段文字有加粗。因此个人并不建议你使用这种字体。


感谢您的反馈，粗体本来是作为 Medium 字重，后因为有用户提议才将 Medium 改为 Bold 字重（lxgw/LxgwWenKai#37）。之后的版本会考虑调整，Bold 将改回 Medium，Light 将改为 SemiLight。
![image](https://user-images.githubusercontent.com/51902309/171519417-e300cce5-5f2c-4839-a40d-80dc68794835.png)

而在此基础上加粗调整的 [臻楷](https://github.com/lxgw/LxgwZhenKai) 依然有很多人觉得细，或许有原字体比较细的原因，主要还是我在字重的选择上太保守了，直接机械加粗又怕糊掉。
